### DISCUSSION TITLE
tags and categories instead of tag and category frontmatter fields

### DISCUSSION THREAD

**Discussion by StefanoCecere** [vuepress-theme-hope/vuepress-theme-hope#4078] [Q&A] [answered]
i know it could be difficult but...
since all markdown editors (see [Obsidian](https://obsidian.md/)) use `tags` and `categories` as frontmatter fields... now i can't use Vuepress Theme Hope without losing the interoperability

what could we do?

PS: Hope would be PERFECT to publish any vault! it's really incredible. much better than Hugo/mkDocs/Quartz...

**Comment by StefanoCecere** [comment]
PS: i see now this [2.0.0-rc.10](https://github.com/vuepress-theme-hope/vuepress-theme-hope/compare/v2.0.0-rc.9...v2.0.0-rc.10)
(2023-12-30)
`The theme now thinks that categories and tags in frontmatter are standard, which will improve experience with tools like Obsidian.`

ok they work in the list.. but not in the single page view

**Comment by Mister-Hope** [comment, marked as answer]
https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/a7d127c6f1e748824f41e9554d51a4f96c69f41b

**Reply by StefanoCecere** [reply]
love you!
